packages/ember-runtime/lib/system/array_proxy.js

```diff
@@ -33,6 +35,40 @@ var EMPTY = [];
 
 function K() { return this; }
 
+export function removeAt(array, start, len) {
+  if ('number' === typeof start) {
```

Comment:
@krisselden to be able to call `removeAt(array, 1, 2);` in some other files such as `arranged_content_test.js`, although there might not be any advantage over `array.removeAt(1, 2);`.